packages/storybook8/stories/Components/VideoGallery/Docs.mdx

```diff
@@ -154,51 +160,68 @@ The vertical gallery can be most useful when the application is running a very w
       src="images/short-screen-share.png"
       alt="Fill to frame in VideoGallery"
     />
-    <>❌ Wide screen share appearence horiztonal gallery</>
+    <>
+      <br />❌ Wide screen share appearence horiztonal gallery
+    </>
   </Stack>
   <Stack horizontalAlign="center">
     <Image
       style={{ width: '100%', height: 'auto' }}
       src="images/short-screen-share-vertical.png"
       alt="Fill frame menu item in VideoGallery"
     />
-    <>✅ Wide screen share appearence vertical gallery</>
+    <>
+      <br />✅ Wide screen share appearence vertical gallery
+    </>
   </Stack>
 </Stack>
 
-## Screen Sharing Experience
+## Content Sharing Experience
 
 The screen shared is the only element placed in the GridLayout and all remote participants are placed in the horizontal gallery in the lower section. To be able to view this screen share, the sharing participant should have their `isScreenSharingOn` prop set to true as well as a defined `screenShareStream` prop (see `localParticipant` and `remoteParticipants` props).
```

Comment:
This is the paragraph for the Content Sharing section on Storybook6  
`Our VideoGallery component is designed for optimal content sharing, incorporating both screen sharing and PowerPoint Live functionalities. The screen shared is the sole element placed in the [Grid Layout](https://60c7ae6891f0e90039d7cd54-wkswzouqpn.chromatic.com/?path=/docs/ui-components-gridlayout--grid-layout), with all remote participants positioned in the horizontal gallery at the lower section. For effective screen sharing or PowerPoint Live, ensure the sharing participant has their isScreenSharingOn prop set to true, along with a defined screenShareStream prop (refer to localParticipant and remoteParticipants props).`

This content seems more related to content sharing while the current one seems to be talking about screen sharing?